deploy/chaos_crds.yaml

```diff
@@ -46,10 +46,8 @@ spec:
               properties:
                 appkind:
                   type: string
```

Comment:
Let us continue to keep the patterns for cases where the fields are populated- while also allowing a empty string, instead of removing them.